packages/ethereum-contracts/contracts/agreements/ConstantFlowAgreementV1.sol

```diff
@@ -140,9 +140,11 @@ contract ConstantFlowAgreementV1 is
          internal pure
          returns (uint256 deposit)
      {
-         require(flowRate > 0, "CFA: not for non-positive flow rate");
-         require(uint256(int256(flowRate)) * liquidationPeriod <= uint256(int256(type(int96).max)),
-             "CFA: flow rate too big");
+        // @note not sure why this was > 0 before, should be allowed to be 0?
```

Comment:
follow up: this is safe to do, it is being used by `getDepositRequiredForFlowRate` which is an external function that has a check: `if (flowRate == 0) return 0;`, so the revert should never be hit if it is 0